examples/generative/diffusion/training_loop.py

```diff
@@ -470,5 +494,6 @@ def training_loop(
             break
 
     # Done.
-    logger0.info()
+    if not dataset == "dfsr":
```

Comment:
Is this needed? Maybe remove `logger0.info()` altogether

Reply:
The ``logger0.info()`` code was skipped because it used to cause syntax error on my computer. This might be due to version mismatch of PythonLogger on my computer. The code ``logger0.info()`` has now been removed.